fronts-client/src/components/form/ArticleMetaForm.tsx

```diff
@@ -530,15 +538,6 @@ class FormComponent extends React.Component<Props, FormComponentState> {
 			);
 		};
 
-		const allowGigaBoost = () =>
```

Comment:
this is no longer required as the renderBoostToggles function manages show / hiding of toggles